esp_encrypted_img/README.md

```diff
@@ -216,11 +216,9 @@ When using an HMAC-derived device key for ECIES-P256, the device hmac key must b
 
     Replace `BLOCK_KEY<N>` with the actual eFuse block you intend to use (e.g., `BLOCK_KEY0`, `BLOCK_KEY1`, etc., up to `BLOCK_KEY5` typically).
 
-2. **Kconfig Configuration for eFuse Block:**
+2. **Code Configuration:**
 
-    The eFuse block chosen in the command above must match the configuration in your project. In the `pre_encrypted_ota` example, this is set using the Kconfig option:
-
-    * `PRE_ENC_OTA_HMAC_EFUSE_KEY_ID`: This integer value (e.g., 0 for `BLOCK_KEY0`, 1 for `BLOCK_KEY1`, etc.) specifies which eFuse key block holds the HMAC key. Ensure this matches the `<N>` used in the `efuse-burn-key` command.
+    The eFuse block chosen in the command above must match the HMAC key ID used in your application code. In the `pre_encrypted_ota` example, the HMAC key ID is defined in the source code and should be updated to match your chosen eFuse block.
```

Comment:
can we also mention the variable or configuration option which needs to be set here for the HMAC key id?

Reply:
Sure, updated the doc